docs/resources.rst

```diff
@@ -71,22 +60,31 @@ Editor Integrations
 
 * Vim
 
-    * `Vim Solidity <https://github.com/tomlion/vim-solidity/>`_
-        Plugin for the Vim editor providing syntax highlighting.
+    * `Vim Solidity by Thesis <https://github.com/thesis/vim-solidity/>`_
+        Syntax highlighting for Solidity in Vim.
+
+    * `Vim Solidity by TovarishFin <https://github.com/TovarishFin/vim-solidity>`_
+        Vim syntax file for solidity.
 
     * `Vim Syntastic <https://github.com/vim-syntastic/syntastic>`_
         Plugin for the Vim editor providing compile checking.
 
 * Visual Studio Code
```

Comment:
```suggestion
* Visual Studio Code (VS Code)
```
While we are at it it probably makes sense to add "VS Code" there since most people use that name for Visual Studio Code, especially if we are referencing VS Code in the list below.